### ISSUE TITLE
CMAF in Ingest

### ISSUE THREAD

**Issue by haudiobe** [Dash-Industry-Forum/Ingest#70]
Clause 6.4 adds unnecessary CMAF specification text and should be removed.

**Comment by RufaelDev** [comment]
CMAF is a closed ISO spec, while CMAF ingest tries be open DASH-IF spec, the text was added for convenience of the reader. I added a reference and a note that CMAF text will prevail in case different. This text is needed to explain the signalling of switchingsets as this was not clear before

**Comment by waqarz** [comment]
> I added a reference and a note that CMAF text will prevail in case different. 

This causes duplication and confusion. CMAF constraints should go to CMAF if needed.

**Comment by boy-vd** [comment]
> CMAF constraints should go to CMAF if needed.

To make the specification legible and useful it should contain enough information so that it is possible to read and understand it as a standalone document. As there has been confusion around the concept and implementation of switching sets before, including this table seems relevant to alleviate possible misunderstandings.

See also: #39.

**Comment by haudiobe** [comment]
I do not believe that a spec that references CMAF can implement CMAF without reading CMAF. We should avoid any duplication and mismatch

**Comment by waqarz** [comment]
@RufaelDev,  @haudiobe, I dont see how commit https://github.com/Dash-Industry-Forum/Ingest/commit/9237b5487b4147d71e19443244d59c6edbe964c6 addresses this issue?

**Comment by RufaelDev** [comment]
lines 967-969 were added
